app/views/layouts/_common_ngapp.html.haml

```diff
@@ -0,0 +1,37 @@
+= render :partial => 'layouts/doctype'
```

Comment:
What differentiates this layout from ```application.html.haml``` is the line below - 
```
%body{:id => id, :onresize => "miqOnResize();", :onload => is_browser_ie? ? '' : 'miqOnLoad();', 'data-controller' => controller_name, "ng-app" => "ManageIQ.angularApplication"}
```
Notice the use of ```ng-app```
Since the other angular forms do not use ```ng-app``` yet, I cannot use```ng-app``` in ```application.html.haml``` -- it would cause the other forms to break.
... and hence the need for a separate layout.